source/authors.md

```diff
@@ -15,17 +15,9 @@ kernelspec:
 
 # About the authors
 
-**Tiffany Timbers** is an Assistant Professor of Teaching in the Department of
-Statistics and Co-Director for the Master of Data Science program (Vancouver
-Option) at the University of British Columbia. In these roles she teaches and
-develops curriculum around the responsible application of Data Science to solve
-real-world problems. One of her favorite courses she teaches is a graduate
-course on collaborative software development, which focuses on teaching how to
-create R and Python packages using modern tools and workflows.
+The original version of this textbook was developed by Tiffany Timbers, Trevor Campbell, and Melissa Lee using the R programming language. The content of R textbook was adapted to Python by Trevor Campbell, Lindsay Heagy, and Joel Ostblom.
```

Comment:
```suggestion
The original version of this textbook was developed by Tiffany Timbers, Trevor Campbell, and Melissa Lee using the R programming language. The content of R textbook was adapted to Python by Trevor Campbell, Lindsey Heagy, and Joel Ostblom.
```

Joel: I am fine as third author if you would like to be second! You have put in a lot of thought and work on this book, so that would be fitting as far as I am concerned

Reply:
All good 😄